Automattic/pocket-casts-android
# Referrals - guest pass card background animation

## Description
This adds guest pass card background animation.

## Testing Instructions
1. Login with a paid account
2. Go to Profile tab
3. Tap gift icon 
4. ✅ Notice that send guest pass screen is shown with an animation in the guest pass  card.

## Screenshots or Screencast 

https://github.com/user-attachments/assets/6173a2ed-ca56-4a1f-949d-7231eed7fcf3

## Checklist
 
#### I have tested any UI changes...
